### DISCUSSION TITLE
VcAjaxBar这个组件的@start和@stop事件没有传递参数过来，我该如何判断某个URL是否加载完毕？

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#571] [Q&A] [answered]
[如题，如果我要监听多个图层url的加载情况时
![1718953651770](https://github.com/zouyaoji/vue-cesium/assets/87740252/18e7fe0e-d966-4f7e-b3c9-b1ce10ee4895)，如何区分加载开始和结束的url？

**Comment by zouyaoji** [comment, marked as answer]
不行哈，这个把控的是整体的，stop代表的是当前所有url都请求完成的事件，即使触发也是拿到最后一个请求完成的url。
